components/src/core/Drawer/Drawer.tsx

```diff
@@ -145,10 +151,13 @@ export type DrawerComponentProps = {
     // Toggles the drawer side border instead of a drop shadow
     sideBorder?: boolean;
 
+    // Drawer variant type
+    variant?: 'persistent' | 'permanent' | 'temporary' | 'rail';
+
     // Sets the width of the drawer (in px) when open
     width?: number;
 } & PXBlueDrawerNavGroupInheritableProperties &
-    Omit<DrawerProps, 'translate'>;
+    Omit<Omit<DrawerProps, 'translate'>, 'variant'>;
```

Comment:
Does this work?

`Omit<DrawerProps, 'translate' | 'variant'>` ?

Reply:
Negative. The `Omit` helper type only allows you to omit a single property.

Reply:
`Pick` let's you specify multiple in that way, but not `Omit` for some reason.

Reply:
Ah sounds good!

Reply:
Actually, it's looking like your suggestion may be possible now...looking into it.

Reply:
It does seem to work...I wonder if that's something they fixed in typescript 4. 

We can fix this in doc-it now:

![image](https://user-images.githubusercontent.com/29152776/104640822-fa46b680-5676-11eb-9511-51dbd037e19b.png)